conf/ocsci/live_deployment_odf_addon.yaml

```diff
@@ -0,0 +1,5 @@
+---
+# This config file can be used for live deployment with ocs addon
+DEPLOYMENT:
+  live_deployment_enable_addon: True
```

Comment:
Please document this value here:
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/README.md

But as I proposed I would rename this file for disable addon and change this name of var for disable purpose.